packages/ts-moose-lib/src/consumption-apis/runner.ts

```diff
@@ -27,6 +28,7 @@ const [
   CLIENT_CERT, // Optional
   CLIENT_KEY, // Optional
   API_KEY, // Optional
+  IS_DMV2,
```

Comment:
is that optional or not?

If not - I am about to PR commander, you might want to rebase on that once merged

Reply:
I merged my PR - please rebase 😃